### PR TITLE
mirrored toffee setup

### PR DESCRIPTION
**Pull request by stravze**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: k8s/environments/dev/common-overlay/video-hearings/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../../../../namespaces/toffee
```
Comment (timja) [inline k8s/environments/dev/common-overlay/video-hearings/kustomization.yaml]:
```suggestion
- ../../../../namespaces/video-hearings
```

File: k8s/environments/dev/dev-helmrelease.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: demo-helmrelease
```
Comment (timja) [inline k8s/environments/dev/dev-helmrelease.yaml:5]:
```suggestion
  name: dev-helmrelease
```

File: k8s/namespaces/video-hearings/kustomization.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: toffee
```
Comment (timja) [inline k8s/namespaces/video-hearings/kustomization.yaml:4]:
```suggestion
namespace: vh
```

File: k8s/environments/dev/common-overlay/kustomization.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- toffee
```
Comment (timja) [inline k8s/environments/dev/common-overlay/kustomization.yaml]:
```suggestion
- video-hearings
```

**Review by timja — COMMENTED** [review]
needs at least these fixes, you can run `kustomize build --load_restrictor none .` from inside the common-overlay folder to verify the output looks right

File: k8s/environments/dev/common-overlay/video-hearings/kustomization.yaml (hunk 1 above)
Comment (stravze) [reply k8s/environments/dev/common-overlay/video-hearings/kustomization.yaml]: sorted

File: k8s/environments/dev/dev-helmrelease.yaml (hunk 2 above)
Comment (stravze) [reply k8s/environments/dev/dev-helmrelease.yaml]: sorted

File: k8s/environments/dev/common-overlay/kustomization.yaml (hunk 4 above)
Comment (stravze) [reply k8s/environments/dev/common-overlay/kustomization.yaml]: sorted

File: k8s/namespaces/video-hearings/kustomization.yaml (hunk 3 above)
Comment (stravze) [reply k8s/namespaces/video-hearings/kustomization.yaml]: sorted

**Review by timja — COMMENTED** [review]
Fails to build, there's a typo somewhere:

```
$ kustomize build --load_restrictor none .
Error: accumulating resources: accumulateFile "accumulating resources from 'video-hearings': read /Users/timja/projects/hmcts/shared-services-flux/k8s/environments/dev/common-overlay/video-hearings: is a directory", accumulateDirector: "recursed accumulation of path '/Users/timja/projects/hmcts/shared-services-flux/k8s/environments/dev/common-overlay/video-hearings': trouble configuring builtin PatchTransformer with config: `\npath: aadpodidentity.yaml\ntarget:\n  group: aadpodidentity.k8s.io\n  kind: AzureIdentity\n  name: vh-aad-identity\n  version: v1\n`: unable to parse SM or JSON patch from [---\napiVersion: \"aadpodidentity.k8s.io/v1\"\nkind: AzureIdentity\nmetadata:\n name: vh-aad-identity\n namespace: vh\nspec:\n type: 0\n resourceID: /subscriptions/867a878b-cb68-4de5-9741-361ac9e178b6/resourceGroups/vh-infra-core-dev/providers/Microsoft.ManagedIdentity/userAssignedIdentities/vh-infra-core-dev-kvuser\n clientID: 988da2df-8e5d-4338-92f4-d3e5fba50eec\n\n---\n\napiVersion: \"aadpodidentity.k8s.io/v1\"\nkind: AzureIdentityBinding\nmetadata:\n name: vh-aad-identity\n namespace: vh\nspec:\n azureIdentity: vh-aad-identity\n selector: vh-aad-identity\n]"
```

File: k8s/namespaces/video-hearings/vh-user-api/vh-user-api.yaml [hunk 5]
```diff
@@ -14,7 +18,3 @@ spec:
       replicas: 2
       useInterpodAntiAffinity: false
       image: sdshmctspublic.azurecr.io/vh/vh-user-api:latest
```
Comment (timja) [inline k8s/namespaces/video-hearings/vh-user-api/vh-user-api.yaml:20]: is this right? it doesn't match our pattern of:

$register/$product/$component

project = team
component = app name

**Review by timja — APPROVED** [review]

File: k8s/namespaces/video-hearings/vh-user-api/vh-user-api.yaml (hunk 5 above)
Comment (stravze) [reply k8s/namespaces/video-hearings/vh-user-api/vh-user-api.yaml:20]: project / team = vh
component / app = vh-user-api

Comment (stravze) [reply k8s/namespaces/video-hearings/vh-user-api/vh-user-api.yaml:20]: Unless i'm misunderstand it ?

Comment (timja) [reply k8s/namespaces/video-hearings/vh-user-api/vh-user-api.yaml:20]: normally it would be:
`sdshmctspublic.azurecr.io/vh/user-api:latest`

Comment (stravze) [reply k8s/namespaces/video-hearings/vh-user-api/vh-user-api.yaml:20]: sorry I see what you mean, now

**Pull request merged**